Mathlib/Algebra/BigOperators/GroupWithZero/Action.lean

```diff
@@ -10,14 +10,12 @@ import Mathlib.Data.Finset.Basic
 import Mathlib.Algebra.Group.Action.Basic
 
 /-!
-# Lemmas about group actions on big operators
+# Lemmas about group with zero actions on big operators
```

Comment:
(a) there is not a single result about groups with zero in this file, so this is a very weird change, and (b) why are you refactoring this file at the same time as adding new results about real and complex numbers? Your PRs are sometimes very hard to understand.

Reply:
The PR is far from ready. I am going to split a lot of things off once it passes CI

Reply:
Does it really show on the queue if it doesn't pass CI?

Reply:
No, definitely not. But hopefully one day it will pass CI, and still it shouldn't show up on the queue at that time since you're planning to split it. Marking as WIP now is preparation for this day (and a hint to people who don't use the queue that they shouldn't look at it)